### PR TITLE
feat: add issue templates for better contributor experience

### PR DESCRIPTION
**Pull request by Asmit2952**
Added issue and PR templates to help contributors file their issues effectively. Easy for maintainers to review issues with added labels.

### CODE REVIEW

**Review by TerryHowe — COMMENTED** [review]
I am not a maintainer on this project, but if I would split the PR and issue templates into separate PRs so one doesn't block the other. This may be better than what is used in the oras CLI, but if I wanted a different format, I'd try to make changes there first.

**Comment by FeynmanZhou** [comment]
@Asmit2952 Thank you for submitting the issue and PR templates to this repo. I think they are helpful. Before we merge it into main, can you enable the issue entry in your folk repo? In this way, we can preview those issue templates and make sure the layout looks good to us.

cc @oras-project/oras-www-maintainers @TerryHowe

File: .github/ISSUE_TEMPLATE/create_issue.yml
Comment (FeynmanZhou) [inline .github/ISSUE_TEMPLATE/create_issue.yml:17]: Oras => ORAS

File: .github/ISSUE_TEMPLATE/report_bug.yml
Comment (FeynmanZhou) [inline .github/ISSUE_TEMPLATE/report_bug.yml:20]: Same as above

**Review by FeynmanZhou — COMMENTED** [review]
Please see my comments above. Thanks.

File: .github/ISSUE_TEMPLATE/create_issue.yml
Comment (Asmit2952) [reply .github/ISSUE_TEMPLATE/create_issue.yml:17]: Cool, will update that

**Comment by Asmit2952** [comment]
> I am not a maintainer on this project, but if I would split the PR and issue templates into separate PRs so one doesn't block the other. This may be better than what is used in the oras CLI, but if I wanted a different format, I'd try to make changes there first.

Thank you, I will make different PRs. Should I also change the issue template in ORAS CLI repository?

**Comment by Asmit2952** [comment]
> @Asmit2952 Thank you for submitting the issue and PR templates to this repo. I think they are helpful. Before we merge it into main, can you enable the issue entry in your folk repo? In this way, we can preview those issue templates and make sure the layout looks good to us.
> 
> cc @oras-project/oras-www-maintainers @TerryHowe

I enabled issue entry, but the templates doesn't seem to appear there as they are not in the main branch? Am I missing something?

**Comment by Asmit2952** [comment]
Hey @FeynmanZhou you can preview the templates by going to Files Changed>View File. That might give you a better idea on how they look.

**Review by FeynmanZhou — APPROVED** [review]
/LGTM

**Pull request merged**
